src/resources/Microsoft.Authorization/policyDefinitions/Audit-AzureHybridBenefit-UnusedResourcesCostOptimization.json

```diff
@@ -0,0 +1,104 @@
+{
+    "name": "Audit-AzureHybridBenefit-UnusedResourcesCostOptimization",
+    "type": "Microsoft.Authorization/policyDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+        "policyType": "Custom",
+        "displayName": "Enable AHUB for eligible VMs",
+        "mode": "All",
+        "description": "Optimize cost by enabling Azure Hybrid Benefit. Leverage this Policy definition as a cost control to reveal Virtual Machines not using AHUB.",
+        "metadata": {
+            "version": "1.0.0",
+            "category": "Cost Optimization",
+            "source": "https://github.com/Azure/Enterprise-Scale/",
+            "alzCloudEnvironments": [
+                "AzureCloud",
+                "AzureChinaCloud",
+                "AzureUSGovernment"
+            ]
+        },
+        "parameters": {
+            "effect": {
+                "type": "String",
+                "metadata": {
+                    "displayName": "Effect",
+                    "description": "Enable or disable the execution of the policy"
+                },
+                "allowedValues": [
+                    "Audit",
+                    "Disabled"
+                ],
+                "defaultValue": "Audit"
+            }
+        },
+        "policyRule": {
+            "if": {
+              "allOf": [
+                {
+                  "field": "type",
+                  "in": [
+                    "Microsoft.Compute/virtualMachines",
+                    "Microsoft.Compute/virtualMachineScaleSets"
+                  ]
+                },
+                {
+                  "equals": "MicrosoftWindowsServer",
+                  "field": "Microsoft.Compute/imagePublisher"
+                },
+                {
+                  "equals": "WindowsServer",
+                  "field": "Microsoft.Compute/imageOffer"
+                },
+                {
+                  "field": "Microsoft.Compute/imageSKU",
+                  "in": [
```

Comment:
This list is constantly changing, and I would suggest you revise this to follow the new practice as implemented in [policy](https://azadvertizerdev.z6.web.core.windows.net/azpolicyadvertizer/ad1eeff9-20d7-4c82-a04e-903acab0bfc1.html), look at how the ImageSKU section is implemented.

Reply:
Thanks. Updated.